syntax = "proto3";
// import "google/protobuf/timestamp.proto";

message LocationMessage {
  int32 id = 1;
  int32 person_id = 2;
  string longitude = 3;
  string latitude = 4;
  string creation_time=5;
  // google.protobuf.Timestamp creation_time = 1;
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
}

message PersonMessage {
  int32 id = 1;
  string first_name = 2;
  string last_name = 3;
  string company_name = 4;
}

service PersonService {
    rpc Create(PersonMessage) returns (PersonMessage);
}
